### DISCUSSION TITLE
Is there a way to delete a project in a workspace?

### DISCUSSION THREAD

**Discussion by FlutterFlowJohn** [AntlerVC/firetable#755] [Support / Q&A] [answered]
Is there a way to delete a project in a workspace?

**Comment by BeeBombshell** [comment]
Hi @FlutterFlowJohn, Rowy doesn't support removing or transferring projects yet. I will convert this discussion to an issue to ensure this feature exists in future versions. Till then, I hope you can wait. Let me know if it's urgent.

**Comment by harinij** [comment, marked as answer]
@FlutterFlowJohn We have added support for deleting projects from the workspace now.
<img width="641" alt="Screen Shot 2022-11-09 at 6 03 49 pm" src="https://user-images.githubusercontent.com/307298/201005280-cfe6c7be-5238-4b8c-aa58-85e8aa8bfdcf.png">
